src/ray/raylet/placement_group_resource_manager_test.cc

```diff
@@ -286,14 +289,14 @@ TEST_F(NewPlacementGroupResourceManagerTest, TestNewIdempotencyWithRandomOrder)
   CheckRemainingResourceCorrect(remaining_resource_instance);
   new_placement_group_resource_manager_->ReturnBundle(bundle_spec);
   // 5. prepare bundle -> commit bundle -> commit bundle.
-  ASSERT_TRUE(new_placement_group_resource_manager_->PrepareBundle(bundle_spec));
+  ASSERT_TRUE(new_placement_group_resource_manager_->PrepareBundles(bundle_specs));
   new_placement_group_resource_manager_->CommitBundle(bundle_spec);
   new_placement_group_resource_manager_->CommitBundle(bundle_spec);
   // 6. check remaining resources is correct.
   CheckRemainingResourceCorrect(remaining_resource_instance);
   new_placement_group_resource_manager_->ReturnBundle(bundle_spec);
   // 7. prepare bundle -> return bundle -> commit bundle.
-  ASSERT_TRUE(new_placement_group_resource_manager_->PrepareBundle(bundle_spec));
+  ASSERT_TRUE(new_placement_group_resource_manager_->PrepareBundles(bundle_specs));
   new_placement_group_resource_manager_->ReturnBundle(bundle_spec);
```

Comment:
Can you add unit tests that cover

(1) `PrepareBundles` return all bundles when one of bundles are failed to be prepared.
(2) `PrepareBundles` succeed when all bundles are succeeded to be prepared.

Reply:
added